--- productization/assets/stylesheets/login.css.erb.orig	2014-10-08 14:34:56.075991633 -0400
+++ productization/assets/stylesheets/login.css.erb	2014-10-30 12:10:45.094392311 -0400
@@ -162,10 +162,9 @@
 #login-screen .form {
   position: absolute;
   bottom: 15%;
-  padding-left: 100px;
-  width: 100%;
-  min-width: 850px;
   right: 0px;
+  width: 0px;
+  min-width: 850px;
   background-color: rgba(0, 0, 0, 0.8);
 }
 
@@ -281,3 +280,6 @@
   height: 180px;
   z-index:1;
   }
+
+fieldset > legend > img {
+  display: none;
