--- login.css.erb.orig	2014-09-29 09:20:30.000000000 -0400
+++ login.css.erb	2014-10-27 09:27:30.431513129 -0400
@@ -162,10 +162,11 @@
 #login-screen .form {
   position: absolute;
   bottom: 15%;
-  padding-left: 100px;
-  width: 100%;
+  /* padding-left: 100px; */
+  width: 10%;
+  right: 0px;
   min-width: 850px;
-  background-color: rgba(255, 255, 255, 0.055);
+  background-color: rgba(0, 0, 0, 0.8);
 }
 
 .ie8 #login-screen .form {
